.travis.yml

```diff
@@ -9,11 +9,17 @@ php:
 
 # faster builds on new travis setup not using sudo
 sudo: false
-
+addons:
+  apt:
+    packages:
+      - beanstalkd
 install:
   - '[[ -z "$CI_USER_TOKEN" ]] || composer config github-oauth.github.com ${CI_USER_TOKEN};'
   - travis_retry composer self-update && composer --version
   - travis_retry composer update --prefer-dist --no-interaction
 
 script:
   - php ./vendor/bin/codecept run
+  - if [[ "$TRAVIS_PHP_VERSION" == "7.3" ]]; then php ./vendor/bin/codecept run --coverage-xml; fi
```

Comment:
Test fails on second run because the number of records is twice as large as expected.

Reply:
Fixed it by checking the initial state instead of assuming it.